src/common/dto/sessions/response/get.session.response.ts

```
@@ -1,4 +1,10 @@
-import { SessionEntity } from '../../../../core/sessions/entities/session.entity';
+import { Session } from '@prisma/client';
```

ESLint

[error] 'Session' is defined but never used.

(@typescript-eslint/no-unused-vars)